File: 30 Days of Code/C++/Day 9: Recursion 3.cpp
#include <bits/stdc++.h>

using namespace std;

// Complete the factorial function below.
int factorial(int n) {
    if(n == 1){
        return 1;
    }
    else {
        int fact = 1;
        for(int i = 1;i<=n;++i) {
            fact *= i;
        }
        return fact;
    }

}

int main()
{
    ofstream fout(getenv("OUTPUT_PATH"));

    int n;
    cin >> n;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    int result = factorial(n);

    fout << result << "\n";

    fout.close();

    return 0;
}

File: 30 Days of Code/Python/Day 9: Recursion 3.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the factorial function below.
def factorial(n):
    if(n==1):
        return 1
    else:
        fact = 1
        i = 0
        for i in range(n):
            i += 1
            fact *= i
        return fact

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    n = int(input())

    result = factorial(n)

    fptr.write(str(result) + '\n')

    fptr.close()
